src/pages/Launchpad/Summary/useSubmit/index.ts

```diff
@@ -1,86 +1,133 @@
 import { useNavigate } from "react-router-dom";
 import { Completed, Network } from "slices/launchpad/types";
 import { SimulContractTx } from "types/evm";
+import { TxContent, isTxResultError } from "types/tx";
 import { useSaveAIFMutation } from "services/aws/aws";
 import { useModalContext } from "contexts/ModalContext";
 import { useGetWallet } from "contexts/WalletContext";
 import { TxPrompt } from "components/Prompt";
 import Account from "contracts/Account";
 import { createEndowment } from "contracts/evm/Account";
-import useCosmosTxSender, { Tx } from "hooks/useCosmosTxSender";
-import { getWasmAttribute } from "helpers";
-import { estimateFee, sendTx as sendEVMTX } from "helpers/evm";
-import { EMAIL_SUPPORT, GENERIC_ERROR_MESSAGE } from "constants/common";
+import { logger } from "helpers";
+import { estimateTx, sendTx } from "helpers/tx";
+import { chainIds } from "constants/chainIds";
+import { GENERIC_ERROR_MESSAGE } from "constants/common";
 import { appRoutes } from "constants/routes";
 import toEVMAIF from "./toEVMAIF";
 import toJunoAIF from "./toJunoAIF";
 
 export default function useSubmit(network: Network) {
   const { wallet } = useGetWallet();
-  const sendTx = useCosmosTxSender();
   const [saveAIF] = useSaveAIFMutation();
   const { showModal, closeModal } = useModalContext();
   const navigate = useNavigate();
 
   async function submit(completed: Completed) {
-    if (!wallet) {
-      return showModal(TxPrompt, { error: "Wallet is not connected" });
-    }
+    try {
+      if (!wallet) {
+        return showModal(TxPrompt, { error: "Wallet is not connected" });
+      }
 
-    if (network === "polygon" /**TODO: && check connected wallet's chainID */) {
-      const tx: SimulContractTx = {
-        to: "0x09266441B8Dc93EE70Dbe27A3612eA6e1116f1F3", //TODO: move to src/contracts/evm
-        from: wallet.address,
-        data: createEndowment.encode(toEVMAIF(completed, wallet.address)),
-      };
-      const { tx: simulated } = await estimateFee(wallet, tx);
-      const hash = await sendEVMTX(wallet, simulated);
-      return alert(hash);
-    }
+      const { chain } = wallet;
+      if (
+        network === "polygon" &&
+        !(chain.chain_id === chainIds.polygon || chain.chain_id === "1337") //polygon local
```

Comment:
Maybe we should add local chain IDs to `const chainIds`? We handle mainnet and testnet IDs already in there, adding local IDs would help.

Reply:
"1337" will be removed once we are interfacing with testnet contracts. 
removes the need to 
```
const chainIDs = IS_TEST ? { }  : IS_LOCAL ? { } : { }
```